### PR TITLE
Enabling timeout for subscription-manager call

### PR DESCRIPTION
**Pull request by jupierce**
@kwoodson ptal

### CODE REVIEW

File: ansible/roles/lib_aos_modules/library/aos_rhsm_repos.py [hunk 1]
```diff
@@ -151,7 +154,7 @@ def needs_update(self):
 
     def check_identity(self):
         """verify that this system is entitled"""
-        rcode, _, _ = self.module.run_command([RHSM_BIN, 'identity'])
+        rcode, _, _ = self.module.run_command(RHSM_ENTRYPOINT + ['identity'])
```
Comment (kwoodson) [inline ansible/roles/lib_aos_modules/library/aos_rhsm_repos.py:157]: Was this bracket intentional?

Comment (jupierce) [reply ansible/roles/lib_aos_modules/library/aos_rhsm_repos.py:157]: Still coming up to speed on python, but yes. RHSM_BIN was just a string; the new RHSM_ENTRYPOINT is a list. So:  
list(RHSM_ENTRYPOINT) + list(['identity']) = list(a combination of both) 

It is this combination that is passed to run_command.

**Comment by jupierce** [comment]
ptal @mwoodson

**Comment by jupierce** [comment]
The ops team decided to avoid this change by forcing all clusters to upgrade to subscription manager > 1.17

**Pull request closed without merging**